pytest_splunk_addon/standard_lib/data_models/Certificates.json

```diff
@@ -1,229 +1,245 @@
+                        },
+                        {
+                            "name": "ssl_subject",
+                            "type": "required",
+                            "comment": "The certificate owner's RFC2253 Distinguished Name."
+                        },
+                        {
+                            "name": "ssl_subject_common_name",
+                            "type": "required",
+                            "comment": "This certificate owner's common name."
+                        },
+                        {
+                            "name": "ssl_subject_email",
+                            "type": "optional",
+                            "comment": "The certificate owner's e-mail address."
+                        },
+                        {
+                            "name": "ssl_subject_email_domain",
+                            "type": "optional",
+                            "comment": "The domain name contained within the certificate subject's email address."
+                        },
+                        {
+                            "name": "ssl_subject_locality",
+                            "type": "optional",
+                            "comment": "The certificate owner's locality."
+                        },
+                        {
+                            "name": "ssl_subject_organization",
+                            "type": "required",
+                            "comment": "The certificate owner's organization."
+                        },
+                        {
+                            "name": "ssl_subject_state",
+                            "type": "optional",
+                            "comment": "The certificate owner's state of residence."
+                        },
+                        {
+                            "name": "ssl_subject_street",
+                            "type": "optional",
+                            "comment": "The certificate owner's street address."
+                        },
+                        {
+                            "name": "ssl_subject_unit",
+                            "type": "optional",
+                            "comment": "The certificate owner's organizational unit."
+                        },
+                        {
+                            "name": "ssl_validity_window",
+                            "type": "required",
+                            "validity": "if(isnum(ssl_validity_window),ssl_validity_window,null())",
+                            "comment": "The length of time (in seconds) for which this certificate is valid."
+                        },
+                        {
+                            "name": "ssl_version",
+                            "type": "optional",
+                            "comment": "The ssl version of this certificate."
+                        }
+                    ],
+                    "child_dataset": []
```

Comment:
SSL child dataset is missing from here but present in the official CIM documentation.

Reply:
It can be found here: https://github.com/splunk/pytest-splunk-addon/pull/842/files/ca4f3da227fb94842c8f1043e2f67ed9c8a76d77#diff-6c6f4c4efb60e4b779e15df7ff49e9612948bf904ee6df5f37a32de9de5897bbR67

Reply:
@harshilgajera-crest can you please check if it is in CIM *.json files?

Reply:
Yes it is present in cim jsons: https://cd.splunkdev.com/EnterpriseSecurity/sa-commoninformationmodel/-/blob/develop/packages/sa-commoninformationmodel/src/main/resources/splunk/default/data/models/Certificates.json#L212